src/app/pages/courses/course/course.component.html

```diff
@@ -1,13 +1,14 @@
-<div class="todo-item row" [ngClass]="calculateStatusClass(course.status)" id="course-{{course.id}}">
+<div class="todo-item row" [ngClass]="calculateStatusClass(course.creationDate)" id="course-{{course.id}}">
```

Comment:
Blue and green borders should be added by Directive.
But top rated courses should be marked using `ngClass`